notebooks/245-oneformer-segmentation/245-oneformer-segmentation.ipynb

```diff
@@ -0,0 +1,406 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ONNX_PATH = Path("oneformer.onnx")
</pre><p>This one is not needed anymore as there is no ONNX conversion step anymore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>